src/services/bundle-descriptor-converter-service.ts

```diff
@@ -65,6 +81,45 @@ export class BundleDescriptorConverterService {
     this.generateBundleYamlDescriptor(bundleDescriptor)
   }
 
+  public processThumbnail(): BundleThumbnailInfo {
+    const thumbnail = this.findThumbnail()
+    const thumbInfo = { path: thumbnail, size: 0 }
+    if (thumbnail === '') {
+      this.thumbnail = {
+        ...thumbInfo,
+        status: ThumbnailStatusMessage.NO_THUMBNAIL
+      }
+      return this.thumbnail
+    }
+
+    const stats = fs.statSync(thumbnail)
+    const size = stats.size / 1024
+    const base64 = Buffer.from(fs.readFileSync(thumbnail)).toString('base64')
```

Comment:
this is not enough, you have to provide the URI prefix like: `data:image/png;base64,<base64>`. Note that you have to set `image/jpeg` for both jpg and jpeg extensions

Reply:
done